tools/promtail.sh

```diff
@@ -236,8 +236,14 @@ spec:
             fieldRef:
               fieldPath: spec.nodeName
         image: grafana/promtail:latest
-        imagePullPolicy: IfNotPresent
+        imagePullPolicy: Always
```

Comment:
@DanCech I know this script is used by grafana.com, just want to check if you had any particular reason for setting `IfNotPresent` vs `Always`

Reply:
Nope, it was just generated from the ksonnet output.  Given that it's using `:latest`, `Always` makes much more sense.